app/components/status_tags/base_component.rb

```diff
@@ -12,6 +12,8 @@ def render?
     end
 
     def call
+      return link_text if status == :checked
```

Comment:
I'm a bit concerned about putting such a general condition in the base class. If we want a special case behaviour that applies to documents only then would it make sense to have a small subclass `StatusTags::DocumentComponent` where `call` is just this line then `super`?

Alternatively, we could possibly define the colour for `:checked` to be transparent, so that we still just call the `govuk_tag` method unconditionally but it doesn't add a background.

Reply:
Can't seem to give a transparent colour to the tag component. Will default to blue. Perhaps best to not even call this component then if we are having to render text instead of the component

Reply:
I've removed the dependency on being inside .govuk-task-list, .app-task-list__item, or .bops-task-accordion__section-status, as we want this to be consistent behaviour.